examples/tls/tls_client.c

```diff
@@ -48,6 +48,11 @@
     #include <wolfssl/certs_test.h>
 #endif
 
+#ifdef OPENSSL_COEXIST
+    #undef SSL_SHUTDOWN_NOT_DONE
+    #define SSL_SHUTDOWN_NOT_DONE WOLFSSL_SHUTDOWN_NOT_DONE 
```

Comment:
No, just switch to using `WOLFSSL_SHUTDOWN_NOT_DONE`. None of this code is needed.